File: Javascript/Arrays/Problem5.js
// Rotate array at once




let arr=[2,3,4,5,1]


let temp=arr[0];

for(let i=0;i<arr.length-1;i++){
  arr[i]=arr[i+1];
  
}

arr[arr.length-1]=temp;

console.log(arr)

File: C++/Arrays/Problem5.cpp

// Rotate array at once 


#include <iostream>
#include<bits/stdc++.h>
using namespace std;



int main() 
{
   int n=5;
   int arr[n]={1,2,3,4,5};
   
   int temp=arr[0];
   
   for(int i=0;i<n;i++){
     arr[i]=arr[i+1];
   }
   arr[n-1]=temp;
  for(int i=0;i<n;i++){
    std::cout << arr[i] << std::endl;
  }
    return 0;
}